app/cli/cmd/repl.go

```
@@ -0,0 +1,1160 @@
+				if len(args) > 0 {
+					res += " " + strings.Join(args, " ")
+				}
+				return res, "\\" + cmdString
+			}
+		}
+	}
+
+	return "", ""
+}
+
+func isFileInProjectPaths(filePath string) bool {
```

golangci-lint (1.64.8)

func `isFileInProjectPaths` is unused

(unused)